cognite_toolkit/_cdf_tk/prototypes/featureflag.py

```diff
@@ -0,0 +1,22 @@
+import os
+from functools import lru_cache
+
+import dotenv
+
+
+@lru_cache(maxsize=128)
```

Comment:
Have no idea if neccessary or the right maxsixe for a boolean, but since i read from dotenv it seems smart to cache?